zsh-proxy.plugin.zsh

```diff
@@ -32,14 +33,15 @@ __check_ip() {
     echo "========================================"
     echo "Check what your IP is"
     echo "----------------------------------------"
-    echo -n "IPIP.NET: "
-    curl https://myip.ipip.net
+    echo -n "IPv4: "
+    curl -4 https://ifconfig.co
     echo "----------------------------------------"
-    echo -n "IP.CN: "
-    curl https://ip.cn
+    echo -n "IPv6: "
+    curl -6 https://ifconfig.co
     echo "----------------------------------------"
-    echo "IP.GS: "
-    curl https://ip.gs
+    echo "Info: "
+    curl -s https://ifconfig.co/json | python -m json.tool
```

Comment:
```suggestion
    curl -s https://api.ip.sb/geoip | python -m json.tool
```

Also, I am not sure if everyone has `python` installed by default.

Reply:
Added python command check